src/renderers/dom/client/syntheticEvents/SyntheticEvent.js

```diff
@@ -160,6 +160,7 @@ assign(SyntheticEvent.prototype, {
     this.dispatchConfig = null;
     this._targetInst = null;
     this.nativeEvent = null;
+    this.target = null;
```

Comment:
Yea, that looks right. Seems like we shouldn't have been setting currentTarget in the c'tor anyway.

> I get why it's not
I just meant that it feels awkward to have to check for `=== 'target'` every time. We could probably move it into the else case below where you have it and then drop the continue.